internal/cmd/run/run.go

```diff
@@ -912,7 +912,7 @@ func ExecuteRunner(runner Runner) error {
 	// Routines.Wait() returns an error created by us so it already has a
 	// stack trace and we don't want to add another one here
 	// nolint: wrapcheck
-	return err
+	return errors.WithMessage(err, "Failed to execute runner")
```

Comment:
IMO this message has more potential to be misleading than to be helpful, because it reads as if we failed to execute the fuzzer which is not necessarily the case, there are many other things which can result in `Runner.Run` returning an error.


```suggestion
	return err
```